### DISCUSSION TITLE
Image viewing on files already uploaded

### DISCUSSION THREAD

**Discussion by Davisa1** [dropzone-ui/dropzone-ui-react#244] [Q&A] [answered]
Hi 

I came across this package and it looks awesome! 

Was wondering if it has the behaviour for viewing images already uploaded and If so how this would be achieved.

**Comment by steffanny-dawson** [comment, marked as answer]
I got same doubt few days ago. 
I think it could be performed by providing a valid `imageUrl` prop to FileItem component.

**Comment by JinSSJ3** [comment]
Hi @Davisa1 , Thanks for your opinion 😄 .

It is correct what @steffanny-dawson commented, FileItem can display the uploaded image if you provide `imageUrl` prop to FileItem component.
This prop must be an string representation like URI, base-64 string or a path in the directory. Wha is still needed is to include is a File object and other info. You can check this codesandbox [example](https://codesandbox.io/s/dropzone-ui-imageurl-to-fileitem-m4801?file=/src/App.js:2058-2066) in order to see a live demo that covers that use case. 

In next release we will launch :rocket: another way to display alredy uploaded files. Hope the solution above works for you 👍 .

**Reply by Davisa1** [reply]
Amazing! You guys are awesome!
